hls4ml/backends/fpga/passes/clone.py

```diff
@@ -79,13 +77,17 @@ def transform(self, model, node):
                     attrs = {'size': np.prod(out_var.shape)}
                     idx = layer.inputs.index(output)
                     layer.inputs[idx] = output + '_cpy' + str(i)
-                clone_layer = model.make_node(
+
+                clone_layer: Clone = model.make_node(
                     Clone,
                     'clone_' + node.name,
                     attrs,
                     [output],
                     [output + '_cpy' + str(i + 1) for i in range(len(output_map[output]))],
                 )
+                for i in range(len(output_map[output])):
+                    key = output + '_cpy' + str(i + 1)
+                    clone_layer.attributes[key].type = node.attributes['result_t']
```

Comment:
This is copying the object reference, not the object, right? Is that fine? One could argue that they should really be the same, but I wanted to check.

Reply:
(I think this is probably good since it would be more confusing to have multiple copies of a named type, so I think I will resolve it soon unless someone objects in the meantime.)